src/vivarium_csu_ltbi/components/observers.py

```diff
@@ -37,13 +39,12 @@ def setup(self, builder):
         self.household_tb_exposure = builder.value.get_value(f'{ltbi_globals.HOUSEHOLD_TUBERCULOSIS}.exposure')
 
     def initialize_previous_state(self, pop_data):
-        population = self.population_view.subview(['alive']).get(pop_data.index)
-        self.population_view.update(pd.Series('', index=population.index, name=self.previous_state_column))
+        self.population_view.update(pd.Series('', index=pop_data.index, name=self.previous_state_column))
```

Comment:
don't know why we were doing this.

Reply:
I hit a bug where we grabbed an untracked person